internal/db/migrations/4_alter_table_add_image_name.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE composes ADD COLUMN image_name VARCHAR NOT NULL DEFAULT '';
```

Comment:
So, when I tried leaving the default value as NULL it creates issues when mapping from the db to a golang struct. I get error messages of can't map NULL to *string. Do you how I could change the `.Scan()` so that we can map from NULL to *string. Everything I could find online recommended using packages to handle typing between the db and the golang structs.

Reply:
jackc/pgx handles this just fine if you change ComposeEntry.ImageName to `*string`